ee/codegen/src/generators/vellum-variable-value.ts

```diff
@@ -178,6 +179,43 @@ class ErrorVellumValue extends AstNode {
   }
 }
 
+class ImageVellumValue extends AstNode {
+  private value: VellumImage;
+
+  public constructor(value: VellumImage) {
+    super();
+    this.value = value;
+  }
+
+  public write(writer: Writer): void {
+    const arguments_ = [
+      python.methodArgument({
+        name: "src",
+        value: python.TypeInstantiation.str(this.value.src),
+      }),
+    ];
+
+    if (this.value.metadata !== undefined) {
```

Comment:
This helper from lodash is useful for checking if something is either null or undefined

```suggestion
    if (!isNil(this.value.metadata)) {
```